extensions/azure/azure-test/src/testFixtures/java/org/eclipse/dataspaceconnector/azure/testfixtures/AbstractAzureBlobTest.java

```diff
@@ -30,39 +33,63 @@
 
 public abstract class AbstractAzureBlobTest {
 
-    protected final String accountName = "account1";
-    protected final String accountKey = "key1";
-    protected final String endpoint = "http://127.0.0.1:10000/" + accountName;
-    protected BlobServiceClient blobServiceClient;
-    protected String containerName;
+    protected final String account1Name = "account1";
+    protected final String account1Key = "key1";
+    protected final String account2Name = "account2";
+    protected final String account2Key = "key2";
+    protected BlobServiceClient blobServiceClient1;
+    protected BlobServiceClient blobServiceClient2;
+    protected String container1Name;
+    protected List<Runnable> containerCleanup = new ArrayList<>();
     protected String testRunId = UUID.randomUUID().toString();
 
     @BeforeEach
     public void setupClient() {
-        containerName = "storage-container-" + testRunId;
+        container1Name = "storage-container-" + testRunId;
```

Comment:
Is there going to be _container2Name_?

Reply:
Good point, renamed to `account1ContainerName` for clarity.